### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (#label-tab)

### ISSUE THREAD

**Issue by Tjoosten** [vl-wbk/vl-woordenboek#198]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[VL. woordenboek | Gerief](http://vlaams-woordenboek.test/willekeurig-woordenboek-artikel)
#### Element path
#label-tab
#### Snippet
`<button class="nav-link" id="label-tab" data-bs-toggle="tab" data-bs-target="#label-tab-pane" type="button" role="tab" aria-controls="label-tab-pane" aria-selected="false" tabindex="-1">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.63 (foreground color: #0d6efd, background color: #e5e7eb, font size: 10.8pt (14.4px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 136.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
